README.md

```diff
@@ -207,6 +207,30 @@ If Doxygen and graphviz are available, the process can also build the documentat
 
 You can also selectively enable/disable other features (e.g., specific plugins you don't care about, or whether or not you want to build the recordings post-processor). Use the --help option when configuring for more info.
 
+### Building on FreeBSD
```

Comment:
Please add some more descriptive description here too, as we do for MacOs. The `pkg` line should be "compacted" a bit too (like what we do above for `yum` and `apt-get`), as there's too many lines now I believe.

Reply:
Done